internal/app/cli/serve.go

```diff
@@ -93,6 +81,40 @@ func Serve(host, port string, opts ServeOptions, repo model.RepoSpec) error {
 	return nil
 }
 
+func createHttpHandler(repo model.RepoSpec, opts ServeOptions) (nethttp.Handler, error) {
+	// create an instance of our handler (server interface)
+	handlerService, err := http.NewDefaultHandlerService(repo)
+	if err != nil {
+		return nil, err
+	}
+
+	handler := http.NewTmcHandler(
+		handlerService,
+		http.TmcHandlerOptions{
+			UrlContextRoot: opts.UrlCtxRoot,
+		})
+
+	// collect Middlewares for the main http handler
+	var mws = getMiddlewares(opts)
+	// create a http handler
+	httpHandler := http.NewHttpHandler(handler, mws)
+	return httpHandler, nil
+}
+
+func createTmcHandler(repo model.RepoSpec, opts ServeOptions) (*http.TmcHandler, error) {
```

Comment:
This function is unused in the code.

Reply:
it's been a month... I don't even remember why it was there